Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1261,9 +1261,9 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
                     diagnosticEngine.emit(Problem(diagnostic: diagnostic))
                     continue
                 }
-                guard let url = ValidatedURL(parsingExact: destination) else {
+                guard let url = ValidatedURL(parsingAuthoredLink: destination) else {
```

Comment:
Since the destination is an authored link we should have been using `init(parsingAuthoredLink:)` instead. The documentation for the `init(parsingExact)` initializer even says so:

> When working with authored documentation links use ``init(parsingAuthoredLink:)`` instead.